lisa/node.py

```diff
@@ -477,15 +470,9 @@ def from_existing(
         environment_name: str,
         base_log_path: Optional[Path] = None,
     ) -> Node:
-        node = Node.create(
-            index=len(self._list),
-            runbook=node_runbook,
-            logger_name=environment_name,
-            base_log_path=base_log_path,
+        return self._extracted_from_from_requirement_7(
```

Comment:
Yep, pretty sure this is not human-generated. Please fix the random naming, then.

Reply:
Is this https://github.com/python-rope/rope? Something else?

Reply:
@glima 
Hi,
No, I really didn't use that tool. However, I use an extension that suggest modifications, I think I shouldn't have used it. I am really sorry. But I really did changes by myself, the extension just suggested minor changes. I am sorry.
For the name as you mentioned, I just didn't know what to write, and I don't know if there is guidelines to follow in naming branches, but I am sorry if the name is not appropriate.

Reply:
@Abdelrahman0W  please remove all function extraction changes, I will have a look on other changes.

Reply:
@squirrelsc 
Okay, I will. Thank you so much